Sources/StreamChat/ChatClient.swift

```diff
@@ -76,6 +76,10 @@ public class ChatClient {
     let channelRepository: ChannelRepository
     
     let channelListUpdater: ChannelListUpdater
+    
+    let memberUpdater: ChannelMemberUpdater
+    
+    let memberListUpdater: ChannelMemberListUpdater
```

Comment:
These should be converted to MemberRepository in the future. Also captured in Notion.